### ISSUE TITLE
Unable to reproduce result from TUTORIAL.md

### ISSUE THREAD

**Issue by wshinigamic** [wvangansbeke/Unsupervised-Classification#60]
Hi,

Thank you for the great work and for making it easy to use your code.

I am trying to reproduce the result according to the steps in TUTORIAL.md but am unable to do so.

First, there seems to be a missing import `import torch` and line 71 needs to be be amended to `model.load_state_dict(checkpoint['model'])`.

After fixing the file, I ran `python tutorial_nn.py --config_env configs/env.yml --config_exp configs/pretext/simclr_stl10.yml` but I am getting the following instead:

> Restart from checkpoint repository_eccv/stl-10/pretext/checkpoint.pth.tar
> Fill memory bank for mining the nearest neighbors (train) ...
> Fill Memory Bank [0/10]
> Mine the nearest neighbors (Top-20)
> Accuracy of top-20 nearest neighbors on train set is 27.46
> Fill memory bank for mining the nearest neighbors (val) ...
> Fill Memory Bank [0/16]
> Mine the nearest neighbors (Top-5)
> Accuracy of top-5 nearest neighbors on val set is 28.92

I tried running scan.py but getting around 30+% accuracy.

Would you know why this is the case? Thanks!
